modules/torch/__init__.py

```diff
@@ -5,6 +5,27 @@
 HUE_STEP = 0.125
 HUE_WHITE = -HUE_STEP # special case understood by update_led and hue_step
 
+HUE_RED = 0
+HUE_GREEN = 0.3333
+HUE_BLUE = 0.6666
+
+# Timings for morse code
+MORSE_DOT = 300
+MORSE_DASH = 3 * MORSE_DOT
+MORSE_WORD = 7 * MORSE_DOT
+
+# Flash patterns for the torch (name, [pattern]), 0 is no flashing
```

Comment:
I wonder if the flash patterns should preserve whatever the currently-configured brightness is, rather than reset it to 100%? And have up and down allow brightness to be altered while flashing, rather than cancelling flash mode entirely?